### PR TITLE
[BETA] hotfix #540: listCollection returns realtime collections according to index 

### PR DESCRIPTION
**Pull request by jenow**
Fix a bug where listCollection were returning all the realtime collections despite the index

### CODE REVIEW

File: lib/api/controllers/readController.js [hunk 1]
```diff
@@ -42,7 +42,13 @@ module.exports = function ReadController (kuzzle) {
       });
     }
 
-    realtimeCollections = kuzzle.hotelClerk.getRealtimeCollections();
+    realtimeCollections = kuzzle.hotelClerk.getRealtimeCollections().filter(function(collection) {
+      if (collection.index === requestObject.index) {
```
Comment (scottinet) [inline lib/api/controllers/readController.js]: nitpicking> can be better written using arrow functions:

```js
realtimeCollections = kuzzle.hotelClerk.getRealtimeCollections()
  .filter(collection => collection.index === modifiedRequestObject.index)
  .map(collection => collection.name);
```

**Comment by benoitvidis** [comment]
+1

**Comment by ballinette** [comment]
+1

**Comment by j33f** [comment]
+1 modulo the nitpick spotted by @scottinet

**Pull request merged**